runtime/mangata-rococo/src/lib.rs

```diff
@@ -881,6 +881,8 @@ impl pallet_elections_phragmen::Config for Runtime {
 	type DesiredMembers = DesiredMembers;
 	type DesiredRunnersUp = DesiredRunnersUp;
 	type TermDuration = TermDuration;
+	type MaxVoters = MaxVoters;
+	type MaxCandidates = MaxCandidates;
```

Comment:
Where is this coming from?

Reply:
The parameter_types block on line 778 is defunct. Please remove it and define MaxCandidates where you have defined max voters.